components/NavBar.tsx

```
@@ -1,106 +1,97 @@
+            <Search className="h-5 w-5" />
           </button>
 
           <button
             type="button"
-            className="relative p-2 cursor-pointer text-gray-600 hover:text-gray-800 hover:bg-gray-100 rounded-lg transition-colors"
+            className="relative cursor-pointer rounded-lg p-2 text-gray-600 transition-colors hover:bg-gray-100 hover:text-gray-800"
           >
-            <Bell className="w-5 h-5 md:w-6 md:h-6" />
-            <span className="absolute top-1 right-1 w-2 h-2 bg-red-500 rounded-full"></span>
+            <Bell className="h-5 w-5 md:h-6 md:w-6" />
+            <span className="absolute top-1 right-1 h-2 w-2 rounded-full bg-red-500"></span>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)